src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,521 @@
+            <NGauge x={155} y={295} />
+
+            <EgtGauge x={155} y={410} />
+
+            <ApuMemos x={370} y={335} />
+        </EcamPage>
+    );
+};
+
+type ComponentPositionProps = {
+    x: number,
+    y: number,
+}
+
+const ApuGen = ({ x, y } : ComponentPositionProps) => {
+    const [apuContactorClosed] = useSimVar('L:A32NX_ELEC_CONTACTOR_3XS_IS_CLOSED', 'Bool', 1000);
+    const [busTieContactor1Closed] = useSimVar('L:A32NX_ELEC_CONTACTOR_11XU1_IS_CLOSED', 'Bool');
+    const [busTieContactor2Closed] = useSimVar('L:A32NX_ELEC_CONTACTOR_11XU2_IS_CLOSED', 'Bool');
+
+    const [apuGenLoad] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD', 'Percent', 500);
+    const [apuGenLoadNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD_NORMAL', 'Bool', 750);
+
+    const [apuGenVoltage] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL', 'Volts', 500);
+    const [apuGenPotentialNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL_NORMAL', 'Bool', 750);
+
+    const [apuGenFreq] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY', 'Hertz', 500);
+    const [apuGenFreqNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY_NORMAL', 'Bool', 750);
+
+    enum apuGenState {STANDBY, OFF, ON}
+    const [currentApuGenState, setCurrentApuGenState] = useState(apuGenState.STANDBY);
+
+    const [apuMasterPbOn] = useSimVar('L:A32NX_OVHD_APU_MASTER_SW_PB_IS_ON', 'Bool', 500);
+    const [apuGenPbOn] = useSimVar('A:APU GENERATOR SWITCH', 'Boolean', 750);
+    const [apuAvail] = useSimVar('L:A32NX_OVHD_APU_START_PB_IS_AVAILABLE', 'Bool', 1000);
+
+    useEffect(() => {
+        if (!apuMasterPbOn && !apuAvail /* FIXME: and apu master switch is valid and apu gen push buttion is valid */) {
+            setCurrentApuGenState(apuGenState.STANDBY);
+        } else if (!(currentApuGenState === apuGenState.STANDBY) && !apuGenPbOn /* FIXME: and apu master switch valid and apu gen pb valid */) {
+            setCurrentApuGenState(apuGenState.OFF);
+        } else {
+            setCurrentApuGenState(apuGenState.ON);
+        }
+    }, [apuMasterPbOn, apuAvail]);
+
+    return (
+        <>
+            <SvgGroup x={x} y={y}>
+                {(apuContactorClosed && (busTieContactor1Closed || busTieContactor2Closed))
+                    && (
+                        <SvgGroup x={42} y={-28}>
+                            <polygon points="0,20 8,5 16,20" className="Circle" />
+                        </SvgGroup>
+                    )}
+                {currentApuGenState !== apuGenState.STANDBY
+                && <rect className="Box" width={100} height={111} />}
+
+                <text
+                    x={50}
+                    y={20}
+                    className={`Center FontNormal
+                    ${currentApuGenState === apuGenState.OFF
+                         || (!(apuGenPotentialNormalRange && apuGenLoadNormalRange && apuGenFreqNormalRange) && currentApuGenState === apuGenState.ON) && ' Amber'}`}
+                >
+                    APU GEN
```

Comment:
**issue**: APU GEN is shown in white when OFF, whereas amber is expected.

![image](https://user-images.githubusercontent.com/1074786/145785612-86c93e7d-55ec-4197-bbc0-7e72e221426d.png)
![image](https://user-images.githubusercontent.com/1074786/145785565-a5672e88-9b5f-40c5-9e44-ea399ad5c174.png)

Reply:
Thank you; addressed.